ceph-deploy/build/setup

```diff
@@ -19,7 +19,7 @@ fi
 
 # Create the virtualenv
 virtualenv $WORKSPACE/venv
-source $WORKSPACE/venv/bin/activate
+VENV="$WORKSPACE/venv/bin"
```

Comment:
Nope, we apparently missed this one in the setup of ceph-deploy. We fixed it in the build script yesterday though.